sycl/source/detail/global_handler.cpp

```diff
@@ -89,6 +91,18 @@ std::mutex &GlobalHandler::getHandlerExtendedMembersMutex() {
   return getOrCreate(MHandlerExtendedMembersMutex);
 }
 
+ThreadPool &GlobalHandler::getHostTaskThreadPool() {
+  static int Size = SYCLConfig<SYCL_QUEUE_THREAD_POOL_SIZE>::get();
+  ThreadPool &TP = getOrCreate(MHostTaskThreadPool, Size);
+
+  {
+    static std::once_flag Flag;
+    std::call_once(Flag, [&] { TP.start(); });
+  }
```

Comment:
Since we are doing this unconditionally maybe we can call start from the `ThreadPool` c'tor to avoid additional locks here?